Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11-slim
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
The `COPY requirements.txt requirements.txt` command should be updated to `COPY requirements.txt .` to ensure the file is placed correctly in the working directory.